embc-app/ClientApp/src/app/volunteer/pages/referral-maker/referral-maker.component.html

```diff
@@ -85,7 +110,12 @@ <h1>Create Referrals</h1>
       </article>
     </section>
 
-    <!-- TODO: prompt for confirmation; disable Finalize until confirmed; set confirmChecked=true -->
+    <div class="mt-5 space-between">
```

Comment:
Please right-align this block and put text on 2 lines as in the mockups.

Also, although this is missing from the mockups, the Finalize Referrals button should be disabled until the confirmation is checked.